woocommerce/woocommerce
# Changelogger: Fix PHPCS violations

### All Submissions:






### Changes proposed in this Pull Request:



Fix PHPCS code violations in the changelogger tool.

### How to test the changes in this Pull Request:

1. `phpcs -s tools/changelogger` or see the action pass on this PR
2. Make sure changelogger still works, ie
```
cd packages/js/components
./vendor/bin/changelogger write
```

### Other information:




### FOR PR REVIEWER ONLY:
